docs/Internal Systems/Getting_started_with_Internal_Sytems_Repo.md

```diff
@@ -0,0 +1,57 @@
+# Getting started with Internal Sytems Repository
+
+This document contains all the information to understand Internal Sytems repository and what you
+need to know to start coding.
+
+## Environment Setup
```

Comment:
This doc should be in the [internal-systems](https://github.com/thoth-tech/Internal-Systems) repository.